scripts/check-runtime.sh

```diff
@@ -69,6 +69,7 @@ if ! has_runtime_changes "origin/${MAIN_BRANCH}" "${GITHUB_SHA}"; then
 Checking Cargo.lock for changes in REFs for $SUBSTRATE_REPO
 EOT
 
+      echo "$(git diff "origin/$MAIN_BRANCH...${GITHUB_SHA}" Cargo.lock | grep -e "$SUBSTRATE_REPO_CARGO" | awk -F '#' '{print $2}')"
```

Comment:
Is this just for debugging? Seems unrelated to the rest of the changes

Reply:
I think that may be a left-over from one of my own PRs where I was debugging this script.

Reply:
That was me adding a new temporary commit to show the broken CI explicitly. I haven't had revisited the PR until now.